src/tutorial/scheduling/datetime-cycling.rst

```diff
@@ -266,6 +266,55 @@ Intercycle Dependencies
 * ``baz[^]``: the task ``baz`` from the initial cycle point.
 
 
+.. _tutorial-clock-triggers:
+
+Clock Triggers
+--------------
+
+.. ifnotslides::
+
+   In Cylc, :term:`cycle points <cycle point>` are just task labels. Tasks are
+   triggered when their dependencies are met, regardless of cycle point.
+   But *clock triggers* can be used to force tasks to wait for a particular
+   real time, relative to their cycle point, before running.
+   This is necessary for certain operational and monitoring systems, e.g. for
+   tasks that process real-time data.
+
+   For example in the following workflow the cycle ``2050-01-01T12Z`` will wait
+   until 12:00 on the 1st of January 2050 before running:
+
+.. code-block:: cylc
+
+   [scheduling]
+       initial cycle point = 2050-01-01T00Z
+       [[xtriggers]]
+           on_time = wall_clock()
+       [[graph]]
+           # "daily" will run, at the earliest, one hour before midday.
+           T12 = @on_time => do_this_on_or_after_noon
+
+.. ifnotslides::
+
+   Clock triggers take an argument - ``offset`` which allows you to specify
+   an offset. For example, it might be ok for a task to run up to an hour
+   before the specified cycle point:
+
+``wall_clock(-PT1H)``
+
+.. ifnotslides::
+
+   All xtriggers take a ``:recurrance`` indicating how often the scheduler
+   will run the check.
+
+   It is good practice not to check more often than necessary:
+
+``wall_clock():PT17M``
```

Comment:
```suggestion
   All xtriggers can be suffixed with ``:interval`` indicating how often the
   scheduler will run the check. The default is 10 seconds.
   
   It is good practice not to check more often than necessary. This will
   depend on the specific use case. For example:

``wall_clock():PT20M``
```
(17 minutes is a weird interval that is triggering a sort of OCD response in me 😄)